app/controllers/application_controller.rb

```diff
@@ -367,6 +367,7 @@ def report_data
     if options.nil? || options[:view].nil?
       model_view = process_params_model_view(params, options)
       @edit = session[:edit]
+      options_from_session(options)
```

Comment:
Generally I consider the idea of "Fixed `report_data` to take `options`(set in get_view) into account" a step back.

Actually we need to prevent using anything from the session here. The report endpoint needs simplifycation and clean up. All the necessary options should come from the request.

The goal is to have this in the API. The has to be stateles so adding more state (session) is a step back.